{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:python:setuptools:58.5.2:*:*:*:*:*:*:*",
	"cpeNameId": "64A32067-53D7-4EA7-8EB4-D43CCA455B5A",
	"created": "2022-12-29T14:39:02.883",
	"lastModified": "2022-12-29T14:43:10.977",
	"titles": [
		{
			"title": "Python setuptools 58.5.2",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://pypi.org/project/setuptools/#history",
			"type": "Version"
		},
		{
			"ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
			"type": "Advisory"
		}
	]
}
